src/debugger_requests.jl

```diff
@@ -822,7 +822,7 @@ function set_variable_request(conn, state::DebuggerState, params::SetVariableArg
 
     if var_ref.kind == :scope
         try
-            ret = JuliaInterpreter.eval_code(var_ref.value, "$var_name = $var_value");
+            ret = JuliaInterpreter.eval_code(var_ref.value, "$var_name = $var_value")
```

Comment:
Shouldn't remove the trailing `;`.

Reply:
does removing it change the semantics of the program?

Reply:
No, but that can't be the criterion, there are countless changes that one could make that don't change the semantics program.

I think we had agreed that the minimal formatter that we want to have in the extension should change whitespacing, indentation and nothing else. In particular, it should never add or remove any other characters that a user wrote.

Obviously this is a great feature to have available in a style and as an option.